components/src/core/components/Input/__tests__/select-input.spec.ts

```diff
@@ -40,6 +40,10 @@ const options = [
 ];
 
 describe('SelectInput.vue', () => {
+  const div = document.createElement('div');
```

Comment:
Due to a bug in jsdom v16 and above, focus is not emitted unless 'attachedTo' value is added to wrapper.  Newer version of vue test utils documentation is missing this. (below exempt is taken from [vue2 docs](https://vue-test-utils.vuejs.org/api/wrapper/#trigger))

refer discussion in https://github.com/vuejs/vue-test-utils/pull/1777

![image](https://user-images.githubusercontent.com/12880059/210499313-797eeb27-c769-43cb-95f6-ba57ec643e00.png)